circle.yml

```diff
@@ -9,11 +9,12 @@ dependencies:
   pre:
     - pip install --upgrade pip
     - pip install Cython --install-option="--no-cython-compile"
-    - pip install numpy==1.11.1 
+    - pip install numpy==1.11.1
     - pip install scipy==0.17.1
     - pip install pytest==3.0.4
+    - pip install flake8==3.3.0
 
 test:
   override:
     - py.test --verbose
-
+    - flake8 ./ --max-line-length=79
```

Comment:
Hmm, perhaps I'm looking at the wrong spot, but it seems that their linting is done in https://github.com/pandas-dev/pandas/blob/master/ci/lint.sh

I'm trying a bit of a dirty method for the moment, but I can revise it to better emulate the approach the pandas team is using.